src/Twig/CommitteeRuntime.php

```diff
@@ -92,10 +93,36 @@ public function isVotingCommittee(Adherent $adherent, Committee $committee): boo
         return $membership && $membership->isVotingCommittee();
     }
 
+    public function getVotingCommittee(CommitteeMembershipCollection $memberships)
+    {
+        $votingCommittee = '';
+
+        foreach ($memberships as $membership) {
+            if ($membership->isVotingCommittee()) {
+                $votingCommittee = $membership->getCommittee()->getSlug();
+            }
+        }
+
+        return $votingCommittee;
+    }
+
     public function isCandidate(Adherent $adherent, Committee $committee): bool
     {
         $membership = $this->committeeManager->getCommitteeMembership($adherent, $committee);
 
         return $membership && $membership->isVotingCommittee() && $membership->getCommitteeCandidacy();
     }
+
+    public function getCandidacy(CommitteeMembershipCollection $memberships)
+    {
+        $committeeName = '';
+
+        foreach ($memberships as $membership) {
+            if ($membership->getCommitteeCandidacy()) {
+                $committeeName = $membership->getCommittee()->getSlug();
```

Comment:
same remarks here for simplifying `foreach` and return Candidacy object if we decide to keep `getCandidacy` as name